config/chime_science_run_gpu.yaml

```diff
@@ -962,6 +978,26 @@ updatable_config:
       ra: 53.6197236741
       dec: 54.6433973569
       scaling: 48
+  bad_inputs:
```

Comment:
Does this not need the additional parameters that are forwarded to receiver kotekan?
```
tag: str
start_time: float
```

Reply:
It does not currently use those (although maybe it should at some point).  However if we want it to match the receiver kotekan then I can add them?

Reply:
Aren't the `bad_inputs` sent meant to be applied in the future?

Reply:
In the RFI system they have always just been applied as they become available.  In other systems they are applied in the future since we want them to be synchronized across the array.  But it this case the bad inputs are just trying to improve the statistics of the RFI SK estimator for a given frequency, so synchronization isn't as important as reacting quickly to bad feed events which might cause an entire frequency to otherwise become zeroed.